LISA/Executor.cpp

```diff
@@ -186,16 +186,16 @@ void Executor::taskRunner(std::function<void()> task)
     INFO(task, " started ");
     try {
         task();
-        // TODO send status "Done"
         INFO(task, " done");
+        operationStatusCallback(currentTask.handle, OperationStatus::SUCCESS, std::string{});
     }
     catch(std::exception& exc){
         ERROR("exception running ", task, ": ", exc.what());
-        // TODO send status "Failed"
+        operationStatusCallback(currentTask.handle, OperationStatus::FAILED, std::string{exc.what()});
     }
     catch(...){
```

Comment:
catch(...) could be removed, all our execption should be derived from std::exception

Reply:
Removed